widget/embedded/src/components/common/ActivateTabModal/ActivateTabModal.tsx

```diff
@@ -0,0 +1,31 @@
+import type { PropTypes } from './ActivateTabModal.types';
+
+import { i18n } from '@lingui/core';
+import { Button, Divider, MessageBox, Modal } from '@rango-dev/ui';
+import React from 'react';
+
+import { getContainer } from '../../../utils/common';
+
+export function ActivateTabModal(props: PropTypes) {
+  const { open, onClose, onConfirm } = props;
+  return (
+    <Modal open={open} dismissible onClose={onClose} container={getContainer()}>
+      <MessageBox
+        title={i18n.t('Activate current tab')}
+        type="warning"
+        description={i18n.t(
+          'Currently, some transactions are running and being handled by other browser tab. If you activate this tab, all transactions that are already in the transaction sign step will expire.'
+        )}>
+        <Divider size={20} />
+        <Button
+          variant="contained"
+          size="large"
+          type="warning"
+          fullWidth
+          onClick={onConfirm}>
+          {i18n.t('Confirm')}
```

Comment:
I think we need to change the color (type).
![Screenshot from 2024-01-16 15-59-22](https://github.com/rango-exchange/rango-client/assets/99180934/0fc67c7f-6c8a-4828-98a5-f5b3113faa40)